docs/developer-docs/multi-chain/ethereum/evm-rpc/evm-rpc-canister.mdx

```diff
@@ -835,6 +837,33 @@ dfx canister call evm_rpc eth_sendRawTransaction "(variant {$RPC_SOURCE}, $RPC_C
 </TabItem>
 </AdornedTabs>
 
+
+### Error messages
+
+#### Error "already known"
+
+```
+"{ code: -32603, message: \"already known\" }"
+```
+
+If you receive this error, but your transaction can be seen on Ethereum, then this error is expected because HTTPS outcalls are replicated meaning only one of the contacted nodes from the JSON-RPC provide should return the success results, while others return an error indicating the transaction is already known.
```

Comment:
I think this error needs a bit more context because it's specific to `eth_sendRawTransaction`, so I'm not sure putting it in a general `Error messages` section is the best place. Generally I think it should mention the following points:

Error when sending a transaction:

1. Sending a transaction to the Ethereum network is a state changing operation and since the EVM-RPC canister sends a transaction to some JSON-RPC providers via HTTPs outcalls , each contacted provider will receive multiple times the same transaction.
2. It is therefore possible that the contacted provider will return an error such as `{ code: -32603, message: "already known" }`, since assuming the sent transaction is valid, one node of the contacted provider should accept the transaction, while the other ones should return some error. Note that the exact error returned is EVM-client specific, since unfortunately the Ethereum JSON-RPC API doesn't specify errors, meaning that other errors could occur or even an error indicating that consensus on the IC could not be reached.
3. Note that even in case an error is returned, you SHOULD assume that your transaction is known to the Ethereum network. You can control whether your transaction was indeed included in a block by querying the transaction count (`eth_getTransactionCount`) at `Latest` block height.

@rvanasa : feel free to expand if you think I forgot something.

Reply:
Follow up PR here: https://github.com/dfinity/portal/pull/3038